src/js/utils.js

```diff
@@ -29,3 +22,72 @@ export function readAssetFile(applicationPath, filename) {
     const assetsDir = path.resolve(__dirname, "../../assets/");
     return fs.readFileSync(path.resolve(assetsDir, applicationPath, filename), "utf8");
 }
+
+/**
+ * Create a list of keys from an array of objects.
+ * @param {Object[]} array - Array of Objects.
+ * @param {string} key - Object attribute to create list for.
+ * @returns {Array}
+ */
+const mapToKeys = (array, key) => {
+    if (!key) return array;
+    return array
+        .map((item) => (lodash.isPlainObject(item) ? item[key] : undefined))
+        .filter(Boolean);
```

Comment:
I don't understand the logic here - maybe an example in the docstring would help?

Reply:
Could we use https://underscorejs.org/#pluck, perhaps??

Might it be that much of the logic re-created anew in this file is already implemented and well-tested in underscore.js?